context-free.md

```diff
@@ -1,22 +1,22 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This seems to be for people who want to make cool art using written instructions (in the form of grammars). This is not a particularly expansive group of people -- probably just people who are bored or trying to make niche abstract-type art.
```

Comment:
Do the features of the language change the intended type of art created or the artist it would entice?